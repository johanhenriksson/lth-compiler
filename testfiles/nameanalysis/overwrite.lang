int main(int b) {
    int b;
    if (b) {
        int b;
    }
}
