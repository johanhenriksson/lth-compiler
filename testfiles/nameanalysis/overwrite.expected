Error at line 2: symbol 'b' is already declared!
